vllm/engine/metrics.py

```diff
@@ -119,7 +119,7 @@ def __init__(self, labelnames: List[str], max_model_len: int):
             buckets=[1, 2, 5, 10, 20],
         )
         self.counter_request_success = Counter(
-            name="vllm:request_success",
+            name="vllm:request_success_total",
```

Comment:
Is this to match the name scheme of the other counters?

Also, can you check if it's used in the example dashboard and, if yes, update the name there too?

Reply:
I updated this to:
- match scheme of other counters
- the dashboard is using `request_success_total` already (so the dashboard was broken)

Reply:
Ah ok, nice.